web/src/frontend/src/digisos/skjema/arbeidUtdanning/Arbeidsforhold.tsx

```diff
@@ -0,0 +1,71 @@
+import * as React from "react";
+import SporsmalFaktum from "../../../nav-soknad/faktum/SporsmalFaktum";
+import SysteminfoMedSkjema from "../../../nav-soknad/components/systeminfoMedSkjema";
+import TextareaFaktum from "../../../nav-soknad/faktum/TextareaFaktum";
+import { finnFakta, getFaktumPropertyVerdi } from "../../../nav-soknad/utils";
+import { State } from "../../redux/reducers";
+import { connect } from "react-redux";
+import { FaktumComponentProps } from "../../../nav-soknad/redux/fakta/faktaTypes";
+import { Faktum } from "../../../nav-soknad/types";
+import Detaljeliste, { DetaljelisteElement } from "../../../nav-soknad/components/detaljeliste";
+import { FormattedMessage } from "react-intl";
+
+const Arbeidsforhold: React.StatelessComponent<FaktumComponentProps> = ({fakta}) => {
+
+	const alleArbeidsforhold: Faktum[] = finnFakta("opplysninger.arbeidsituasjon", fakta);
+
+	return <SporsmalFaktum faktumKey="opplysninger.arbeidsituasjon">
+		<SysteminfoMedSkjema>
+			<fieldset className="skjema-fieldset">
```

Comment:
Ikke bruk fieldset her. Fieldset brukes for å innkapsle et område av et skjema, og her er det bare en tekst.

Reply:
Ok. Har endret til ```<h4 className="skjema-sporsmal__infotekst__tittel">```